switchapi/es2k/switch_lag.c

```diff
@@ -0,0 +1,472 @@
+/*
+ * Copyright (c) 2023 Intel Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at:
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "switchapi/switch_lag.h"
+
+#include "bf_types.h"
+#include "switch_pd_utils.h"
+#include "switchapi/es2k/switch_pd_lag.h"
+#include "switchapi/switch_base_types.h"
+#include "switchapi/switch_device.h"
+#include "switchapi/switch_internal.h"
+#include "switchapi/switch_status.h"
+
+/*
```

Comment:
If you're going to use Doxygen comments, the comment block needs to begin with `/**`, not `/*`.

Reply:
Addressed.